src/components/ExportCombinedPopUp.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <pop-up class="export-popup" title="Exporteer" @close="$emit('close')">
+    <form class="export-popup__content export-popup__form-columns">
+      <fieldset class="export-popup__notification export-popup__notification--loading" v-if="!eeLayer">
```

Comment:
Semantically i believe this should no be a `fieldset` element, but correct me if I'm wrong.

Also, I never see this message appearing in the UI when i make an export, is the `v-if` correct?